tests/unit_tests/CMakeLists.txt

```diff
@@ -107,65 +78,32 @@ endif()
 ###############################################################################
 # ROCSHMEM DEPENDENCY
 ###############################################################################
-find_package(hip REQUIRED)
+find_package(hip REQUIRED HINTS ${ROCM_PATH} $ENV{ROCM_PATH} PATHS /opt/rocm)
 
 if (BUILD_TESTS_ONLY)
   find_package(MPI REQUIRED)
+  find_package(rocshmem REQUIRED HINTS $ENV{ROCSHMEM_HOME} ${ROCM_PATH} $ENV{ROCM_PATH} PATHS /opt/rocm)
```

Comment:
We are phasing out ROCSHMEM_HOME, if it has had no significant uses I would straight up remove it in favor of rocshmem_ROOT.